pages/gallery.html

```diff
@@ -5,8 +5,113 @@
 <body>
-  
+  <section id="gallery-container">
+    <div id="blue">
+      <nav id="side-bar" class="nav flex-column">
+        <a class="nav-link" href="../index.html">Home</a>
+        <a class="nav-link" href="./facts.html">Facts</a>
+        <a class="nav-link disabled">Gallery</a>
+      </nav>
+    </div>
+    <div id="yellow">
+      <header id="gallery-header">
+        <h1>Gallery</h1>
+        <h2>Chewie &#10024; Cecil &#10024; Rhubarb &#10024; Comet</h2>
+      </header>
+    </div>
+    <div id="red">
+      <section id="pic-box">
+        <div class="gallery">
+          <a target="_blank" href="../images/chewie-red-sweater.jpg">
+            <img src="../images/chewie-red-sweater.jpg" alt="Chewie in a red sweater">
+            <p>Chewie in his red sweater</p>
+          </a>
+        </div>
+        <div class="gallery wide">
+          <a target="_blank" href="../images/comet-sleeping.jpeg">
+            <img id="banana" src="../images/comet-sleeping.jpeg" alt="Comet taking a little nap">
+            <p>Comet taking a little nap</p>
+          </a>
+        </div>
+        <div class="gallery">
+          <a target="_blank" href="../images/rhu-and-cecil-window.jpg">
+            <img src="../images/rhu-and-cecil-window.jpg" alt="Two fluffy cats at a window">
+            <p>Rhubarb and Cecil vibing at a window</p>
+          </a>
+        </div>
+        <div class="gallery">
+          <a target="_blank" href="../images/bowtie-chewie.jpg">
+            <img src="../images/bowtie-chewie.jpg" alt="Happy dog wearing bowtie">
+            <p>Chewie smiling in a bowtie</p>
+          </a>
+        </div>
+        <div class="gallery">
+          <a target="_blank" href="../images/comet-selfie.png">
+            <img src="../images/comet-selfie.png" alt="A selfie of Comet judging you">
+            <p>Comet questioning your life choices</p>
+          </a>
+        </div>
+        <div class="gallery wide">
+          <a target="_blank" href="../images/cat-banana.jpeg">
+            <img id="banana" src="../images/cat-banana.jpeg" alt="Cat guarding a banana">
+            <p>Cecil guarding bananas</p>
+          </a>
+        </div>
+        <div class="gallery">
+          <a target="_blank" href="../images/happy-comet.jpeg">
+            <img src="../images/happy-comet.jpeg" alt="Happy dog!">
+            <p>Happy Comet!</p>
+          </a>
+        </div>
+        <div class="gallery wide">
+          <a target="_blank" href="../images/rhubarb-fish.jpeg">
+            <img id="banana" src="../images/rhubarb-fish.jpeg" alt="Cat holding fish toy cutely">
+            <p>Rhubarb with a fish</p>
+          </a>
+        </div>
+        <div class="gallery">
+          <a target="_blank" href="../images/bombastic-side-eye.jpg">
+            <img src="../images/bombastic-side-eye.jpg" alt="Dog giving side eye">
+            <p>Chewie giving some serious side eye</p>
+          </a>
+        </div>
+        <div class="gallery wide">
```

Comment:
I was expecting in the styles to see different rules for `gallery` and `wide` classes but they only seem to be used like
```css
.gallery.wide { }
```
which works perfectly well, but usually using multiple classes implies that each added class has some different rules.